releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -5,7 +5,10 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
 ### v6.0.0 - February, 02 2023
-* Updated the task to require Node.js 16. It previously required 10.
+* **Breaking changes**:
+  - Updated the task to require Node.js 16. It previously required 10.
+  - Updated the minimum supported azure pipeline agent version to [2.206.1](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.206.1) or later. Previously it was [2.144.0](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.144.0).
+
 * Fixed a bug that caused an invalid secret URL warning when resolving key vault references in sovereign clouds.
```

Comment:
Is this a breaking change? If not, a new section header is needed to separate it from **Breaking changes**, such as **Bug fixes**

Reply:
Yes its not a breaking change, added new section header "Bug fixes"